setup.cfg

```diff
@@ -3,10 +3,12 @@ universal = 1
 
 [flake8]
 ignore =
+    # E203: Whitespace before ':'; doesn't work with black
+    E203,
```

Comment:
Black adds whitespace before `:` characters in slices. e.g. `foo = [len(something) :5]`.